lib/adjacent_bomb_count.ex

```diff
@@ -0,0 +1,52 @@
+defmodule AdjacentBombCount do
+  def update_adjacent_bomb_counts(board) do
```

Comment:
The term "update" in here seems a little odd to me. Isn't adjacent bomb count immutable? Once it's been calculated for each space on the board, there's nothing that will change it, right?

Reply:
@ecmendenhall yes, you're right. `set_adjacent_bomb_counts` would make more sense. I'll change it in my next PR! Thank you!